### PR TITLE
Feature/maas 1320 doa checks

### PR DESCRIPTION
**Pull request by pnuckowski**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -65,3 +65,7 @@ ChangeLog
 
 # Vagrant
 .vagrant
+
+*-local.cfg
+.DS_Store
+.venv
```
Comment (deejay1) [inline .gitignore]: new line :P

File: alamo_scheduler/conf.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        return cls._inst
+
+    def __getattr__(self, name):
+        return None
+
+
+settings = AlamoSettings()
+
+
+def initialize_settings(config_data: list):
```
Comment (deejay1) [inline alamo_scheduler/conf.py]: remind me again why do we have to do this?

File: alamo_scheduler/scheduler.py [hunk 3]
```diff
@@ -1,93 +1,125 @@
...
 
-from alamo_scheduler.fixtures.checks import (
-    load_data_to_memory,
-    generate_check_def
-)
-from alamo_scheduler.loggers import logger
+from kafka.client import KafkaClient
+
+from kafka.consumer.simple import SimpleConsumer
+
```
Comment (deejay1) [inline alamo_scheduler/scheduler.py]: looks like these empty lines could go away..

**Comment by deejay1** [comment]
:+1:

**Comment by ch40s3k** [comment]
:+1:

**Pull request merged**